src/components/helpers/ajax.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+class Ajax {
+  fetchTags () {
+    return axios.get('http://localhost:3001/api/v1/tags.json')
```

Comment:
Right this should be a `process.env.DEV_API_URL`

and then in prod: `process.env.PROD_API_URL`